Removes watchOS target and scheme

surpher opened the pull request:
watchOS does not have a test target option so it does not make sense to provide `pact-consumer-swift` for it.

PR will resolve #54 

- Removes watchOS target and scheme
- Introduces a script that checks for `pact-mock-service` before running the tests and fails the build with a message if mock service is not installed
- Cleans up pre- and post- script commands for testing

surpher committed:
Add script to check fro dependencies

surpher committed:
Remove watchOS target and scheme

surpher committed:
Remove unnecessary pre- and post- script commands

surpher committed:
Moves linting step before compiling

andrewspinks commented on PactConsumerSwift.xcodeproj/project.pbxproj:
```diff
@@ -729,7 +675,7 @@
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint\nelse\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi";
+			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint --strict\nelse\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
```
If we are adding `--strict` here, should we also do it in the build script? https://github.com/DiUS/pact-consumer-swift/blob/master/scripts/build.sh#L4

Do we even need it to be run in both places?

surpher replied:
Removed from this PR and will be addressed on a separate branch and PR.

Commit:
Remove swiftlint —strict argument

Commit:
Display Error in Issue Navigator when mock service not installed

Fails the test build and displays error in Issue Navigator when pact-mock-service is not installed.
